lib/rails_response_dumper/colorize.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+COLORS = {
+  red: 31,
+  cyan: 36,
+  green: 32
+}.freeze
+
+def colorize_red(text)
+  colorize(text, :red)
+end
+
+def colorize_cyan(text)
+  colorize(text, :cyan)
+end
+
+def colorize_green(text)
+  colorize(text, :green)
+end
+
+def colorize(text, color)
+  "\e[#{COLORS[color]}m#{text}\e[0m"
```

Comment:
IIUC, this doesn't detect the presence of a TTY so will put these control characters in output logs. An easy way to test this is to pipe the output to a file then look at it in an editor:

```
bundle exec rails-response-dumper > out.txt
```

The file should not contain the control characters.

Reply:
added TTY checking in [Add optional colorization for TTY environments](https://github.com/Pioneer-Valley-Books/rails-response-dumper/pull/46/commits/d518ea8ebb288405d94dcafa37596d944e397051)

Reply:
also moved the output from stderr to stdout, because piping example above made me notice that we miss the actual error details in a pipe with the current setup. 

I wanted to explicitly draw attention to that, because I'm not quite sure if moving to stdout is a good move here or not. I like all of the details going into one stream because it allows you to suppress the console output (if we wanted to add a flag for that later), but it feels strange to move away from stderr.

if that change makes sense, I'll pull it out into a separate PR.

Reply:
great, pulled out in #50